src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useEffect } from 'react';
+import { Range } from 'react-range';
+
+import useTheme from '../../hooks/useTheme';
+import { TestProps } from '../../utils/types';
+import TextField from '../TextField';
+import SliderMark from './components/SliderMark';
+import SliderThumb from './components/SliderThumb';
+import SliderTrack from './components/SliderTrack';
+import { Container, InputContainer, InputsContainer } from './Slider.style';
+
+interface Props {
+  /** Determines the position of the rendered thumbs. Defaults to [0, 100] */
+  values: number[];
+  /** Determines if the Slider is disabled or not */
+  disabled?: boolean;
+  /** Determines whether the Slider is a Range or a Selector */
+  isSelector?: boolean;
+  /** Determines if the Slider will show increments along its track */
+  hasIncrements?: boolean;
+  /** Use this to change the slider's initial values (e.g. [80, 100]) */
+  initialValues?: number[];
+  /** Called when a thumb is moved, provides new values */
+  onChange: (values: number[]) => void;
+  /** Called when a change is finished (mouse/touch up, or keyup), provides current values.
+   * Use this event when you have to make for example ajax request with new values. */
+  onFinalChange?: (values: number[]) => void;
+}
+
+const STEP = 1;
+export const STEP_WITH_INCREMENTS = 20;
+export const MIN = 0;
+export const MAX = 100;
+
+const Slider: React.FC<Props & TestProps> = ({
+  values,
+  initialValues = [MIN, MAX],
+  onChange,
+  onFinalChange,
+  isSelector = false,
+  hasIncrements = false,
+  disabled = false,
+  dataTestPrefixId,
```

Comment:
This is not declared in the props type. Also, IMO, it would make more sense to call it `dataTestIdPrefix`

Reply:
This is from the `TestProps` type that we import in ictinus' components in order to not declare a `dataTestId` prop every time :D